### ISSUE TITLE
Is there currently a search using multiple tag fields at once? 

### ISSUE THREAD

**Issue by SigHunter** [fooyin/fooyin#271]
Hi, 

I currently use a SearchBar in Fooyin that is connected to a bunch of library filter widgets. 
It works, but it is not possible to search for multiple tags at once, 
e.g. combine artist + album in one search query. I can only search for an artist and use the filters to put it in a playlist

For example with foo_quicksearch in Foobar2000 I can search within multiple tag fields, e.g. search artist and album in one search query, upon pressing enter it throws everything that would be found in the library in the new playlist. 
I'd prefer that, so I can get rid of the libraryfilters, to make more screen space for the playlist. in foo_quicksearch it's also possible to customize which tags will be searched, e.g. %album artist% %title% %album%

I noticed there are a bunch of search related items on the [roadmap](https://github.com/fooyin/fooyin/blob/master/ROADMAP.md#fooyin-roadmap).  Is any one of those that? I know query-based language search is different (also nice to have). Is this what metadata searching means? 

Thanks for Fooyin!

**Comment by ludouzi** [comment]
Yeah, the current search functionality is very limited and it only performs simple matching of the entire search string.

Once 0.7.0 is released (very soon), the next feature I'll be prioritising is searching. Adding an entire library search like foo_quicksearch will be very simple as it will use a lot of the same functionality as playlist searching.

Metadata searching on the roadmap refers to internet lookup. I'll make that a bit clearer.

**Comment by SigHunter** [comment]
I see, thanks for clearing that up!

**Comment by SigHunter** [comment]
the mentioned commits made what I was looking for possible. everything is well, thank you very much. case closed
